dockers/docker-teamd/supervisor-proc-exit-listener

```diff
@@ -0,0 +1,60 @@
+#!/usr/bin/env python
+
+import os
+import signal
+import sys
+import syslog
+import swsssdk
+
+from supervisor import childutils
+
+# Contents of file should be the names of critical processes (as defined in
+# supervisor.conf file), one per line
+CRITICAL_PROCESSES_FILE = '/etc/supervisor/critical_processes'
+
+# This table in database contains the features for container and each 
+# feature in a row will be configured a state or number.
+TABLE_NAME = 'CONTAINER_FEATURE'
+
+CONTAINER_NAME = 'teamd'
```

Comment:
Suggest to move this to be the first constant defined in the file. That will make it stand out and easy for someone to notice that it needs to be updated (like if someone adds a new container).

Reply:
Moved the constant string (CONTAINER_NAME) to be the very first variable.